jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostDFSuite.scala

```diff
@@ -138,6 +138,19 @@ class XGBoostDFSuite extends FunSuite with PerTest with TableDrivenPropertyCheck
     assert(xgbEstimatorCopy.fromParamsToXGBParamMap("objective").toString === "binary:logistic")
   }
 
+  test("checkpoint parameters configured correctly") {
```

Comment:
we generally do not recommend have a separate test case for parameters like checkpoint_interval, would you please have a test case which runs with a specified checkpoint_interval and check if the checkpoint file is eventually there?

Reply:
@CodingCat Thx, I changed the test.